### ISSUE TITLE
Query not found

### ISSUE THREAD

**Issue by WuRoc** [prestodb/presto#18237]
UI Today, 2022.8.26, I got Query not found at http://localhost:8080/ui/query.html?20220822_095421_00014_rkm2s a few days ago. How does presto find all the queries, can you summarize the sql of the query, and where is the entry for parsing the sql?

**Comment by WuRoc** [comment]
？？？

**Comment by cliandy** [comment]
Not sure if this is what you're asking, but presto has a maximum history set with `query.max-history` so it may have been removed if enough queries were run

**Comment by WuRoc** [comment]
> Not sure if this is what you're asking, but presto has a maximum history set with `query.max-history` so it may have been removed if enough queries were run

yes，what you said can solve my problem thanks.
